device-types/FS/FMU-MC042733.yaml

```diff
@@ -0,0 +1,30 @@
+---
+manufacturer: FS
+model: FMU-MC042733
+slug: fmu-mc042733
+comments: '[4 Channels 1270-1330nm, LC/UPC, Dual Fiber, Low Insertion Loss CWDM Mux Demux, FMU Plug-in Module](https://www.fs.com/products/42972.html)'
+part_number: '42972'
```

Comment:
According to the website, the FS P/N is `FMU-MC042733`. I see `#42972` in reference to the FS product page.. Since the FS part number is actually `FMU-MC042733` I think your file should reflect this

Reply:
Look at the Picture from them: left side same modelname and left side this part number. Its correct so, its confused me too ;-)

Reply:
Yup I see what you mean, okay lets roll with that